owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/history/util/IDifferenceHandler.java

```diff
@@ -0,0 +1,15 @@
+package fr.openwide.core.jpa.more.business.history.util;
+
+import java.util.Date;
+
+import fr.openwide.core.jpa.more.business.difference.model.Difference;
+
+/**
+ * Met à jour un objet en fonction des différences constatées sur cet objet.
+ * <p>Se résume pour l'instant à simplement rafraîchir certains {@link HistoryEvent}.
+ */
+public interface IDifferenceHandler<T> {
```

Comment:
Mmmh, this interface was only used to refresh HistoryEvents, which are not part of the current implementation. I'm wondering if we should try to reintroduce the HistoryEvents or get rid of this interface.